linux-4.4.defconfig

```diff
@@ -76,6 +76,19 @@ CONFIG_SYN_COOKIES=y
 # CONFIG_INET_LRO is not set
 # CONFIG_INET_DIAG is not set
 # CONFIG_IPV6 is not set
+CONFIG_BT=m
+CONFIG_BT_RFCOMM=m
+CONFIG_BT_RFCOMM_TTY=y
+CONFIG_BT_BNEP=m
+CONFIG_BT_BNEP_MC_FILTER=y
+CONFIG_BT_BNEP_PROTO_FILTER=y
+CONFIG_BT_HIDP=m
+CONFIG_BT_HCIBTUSB=m
```

Comment:
`lsmod` listed the following modules: 
* `hci_uart`
* `btbcm`
* `bnep`
* `bluetooth`

I enabled nearly all BT relevant configurations within the kernel during my debugging sessions. But I never reduced them afterwards to the relevant minimal set.